Pollo/Views/Cards/MCResultCell.swift

```diff
@@ -150,75 +137,107 @@ class MCResultCell: UICollectionViewCell {
             make.width.height.equalTo(selectedImageViewLength)
             make.center.equalTo(dotView)
         }
+        
+        containerView.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(horizontalPadding + (userRole == .admin ? 0 : horizontalPadding + dotViewLength))
+            make.trailing.equalToSuperview().inset(userRole == .admin ? adminTrailingPadding : horizontalPadding)
+            make.height.equalTo(containerViewHeight)
+            make.bottom.equalToSuperview()
+        }
+        
+        optionLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(horizontalPadding)
+            make.trailing.equalToSuperview().inset(horizontalPadding)
+            make.centerY.equalToSuperview()
+            if showCorrectAnswer {
+                make.width.equalTo(optionLabelWidth >= maxWidth ? maxWidth : optionLabelWidth)
+            } else {
+                make.width.equalTo(maxWidth)
+            }
+        }
 
         highlightView.snp.makeConstraints { make in
             make.leading.top.bottom.equalToSuperview()
             make.width.equalToSuperview().multipliedBy(percentSelected)
         }
+        
+        numSelectedLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().inset(numSelectedLabelTopPadding)
+            make.trailing.equalToSuperview().inset(numSelectedLabelTrailingPadding)
+            make.width.equalTo(numSelectedLabelWidth)
```

Comment:
do we need to set the width here and for `percentSelectedLabel`? can you just set leading = containerView and trailing = superView with an offset on either side?
just because we set the containerView's trailing to be inset from the superview with `adminTrailingPadding` so there should be enough space for the `numSelectedLabel` and `percentSelectedLabel`, it just seems redundant to have all these constants for the widths